integration_tests/integration_test.go

```diff
@@ -642,14 +642,15 @@ var _ = Describe("Broker integration tests", func() {
 
 			By("checking if instance uses enterprise cluster", func() {
 				deploymentName := fmt.Sprintf("%s-%s", cfg.DBPrefix, instanceID)
-				deployemnt, errs := composeClient.GetDeploymentByName(deploymentName)
+				deployment, errs := composeClient.GetDeploymentByName(deploymentName)
 				Expect(errs).To(BeNil())
-				clusterURL, err := url.Parse(deployemnt.Links.ClusterLink.HREF)
+				clusterURL, err := url.Parse(deployment.Links.ClusterLink.HREF)
 				Expect(err).ToNot(HaveOccurred())
 				splitPath := strings.Split(strings.TrimRight(clusterURL.Path, "{"), "/")
 				clusterID := splitPath[len(splitPath)-1]
-				cluster, errs := composeClient.GetCluster(clusterID)
+				cluster, errs := composeClient.GetClusterByName(cfg.ClusterName)
```

Comment:
Minor detail: I think it would be easier to follow this if this variable was named `expectedCluster`. It took me a couple of reads to get which cluster was which in the assertion below.